src/ops/mod.rs

```diff
@@ -204,7 +206,8 @@ pub mod test {
                 .as_local()]
                     .borrow_mut()
                     .inner
-                    .on_input(src, u.into(), &self.nodes, &self.states);
+                    .on_input(src, u.into(), &self.nodes, &self.states)
```

Comment:
Updated this to explicitly handle `Base::on_input` returning `None` -- instead of implicitly expecting it to return Some(empty vector).